crates/burn-tensor/src/tests/ops/all.rs

```diff
@@ -0,0 +1,46 @@
+#[burn_tensor_testgen::testgen(all_op)]
+mod tests {
+    use super::*;
+    use burn_tensor::{Data, Tensor};
+
+    #[test]
+    fn test_all() {
```

Comment:
I think we should also have versions of this test that give true. Otherwise the op could just be `return false`